src/class/IrcConnect.php

```diff
@@ -237,8 +245,9 @@ public function sanitizeMsg($rawMsg)
         if ($isMod) array_push($roles, Message::$ROLE_MOD);
         if ($isSub) array_push($roles, Message::$ROLE_SUB);
         if ($isVip) array_push($roles, Message::$ROLE_VIP);
+        if ($hasTurbo) array_push($roles, Message::$TURBO_USER);
 
-        $message = new Message($rawMsg, $id, $this->removeReturns($username), $this->removeReturns($message), $roles);
+        $message = new Message($rawMsg, $id, $this->removeReturns($username), $this->removeReturns($message), $senderId, $roles);
```

Comment:
Possible to sanitize return like for $username ? $this->removeReturns($senderId)

Reply:
Surely, it can be sanitized. As I saw user-id always returns integer so I didn't wrapped it with removeReturns method